packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts

```diff
@@ -10,11 +10,14 @@ export enum TelemetryPropertyKey {
   errorCode = "error-code",
   errorMessage = "error-message",
   validationResult = "validation-result",
+  success = "success",
 }
 
 enum TelemetryPropertyValue {
   UserError = "user",
   SystemError = "system",
+  success = "yes",
```

Comment:
Why the value is not `success` and `fail`?

Reply:
It's aligned with other plugin telemetries

Reply:
@dilin-MS The property name is "success", it's like asking if the operation is succeeded, so the answer should be "yes" or "no".

Reply:
What about `isSuccessful` as property name and `true` / `false` as possible value so as to increase code readability?